Backpack/Rating/Classes/BPKRating.swift

```diff
@@ -234,6 +237,14 @@ public class BPKRating: UIView {
         }
     }
 
+    private func updateTitleSubtitleStackViewSubviews() {
+        let titleViewUpdated = titleSubtitleStackView.arrangedSubviews != [titleView, subtitleLabel]
```

Comment:
I think this boolean represents if the stack needs to be updated, rather than if it was updated. can you rename it to reflect this?

Reply:
I updated it in this commit 6a552388575a0253b740462a2c7bf4006b24fab9